domain/src/main/java/org/mobilitydata/gtfsvalidator/domain/entity/gtfs/trips/WheelchairAccessibleStatus.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2020. MobilityData IO.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.mobilitydata.gtfsvalidator.domain.entity.gtfs.trips;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public enum WheelchairAccessibleStatus {
```

Comment:
Same remark, see if this can be implemented once and reused in multiple GTFS entities

Reply:
Implementing a "generic" WheelchairAccessibleStatus in #100